Makefile

```diff
@@ -0,0 +1,17 @@
+compose_build:
+	docker-compose build
+
+test_db:
+	docker-compose run --rm postgres psql -h postgres -U postgres -c "create database tests"
+
+create_database:
+	docker-compose run server create_db
+
+bundle:
+	docker-compose run server bin/bundle-extensions
+
+test:
+	docker-compose run server tests
+
+build: bundle
+	npm run build
```

Comment:
These tasks needs to be marked as [PHONY](https://stackoverflow.com/a/2145605/153296) to prevent Make from skipping over the steps in case it finds file with the task names, e.g.:

```Makefile
.PHONY: build bundle compose_build create_database test test_db 

# ...
```

Reply:
Any chance to add a clean task? That's a common task added to Makefiles and is often super useful to blow away everything.